### DISCUSSION TITLE
.NET Standard

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/2801#70] [Perguntas e Respostas] [answered]
Num contexto legado (.NET Framework e Core), o .NET Standard é um contrato que ambas as versões precisam implementar.

Então somente um conjunto de bibliotecas básicas e específicas podem ser utilizadas num projeto que usa as duas versões do framework ao mesmo tempo?

**Comment by andrebaltieri** [comment, marked as answer]
Exatamente @PolarisTDM... o .NET Standard garante que tudo que está usando funciona em ambos Frameworks.
